add footer

fickryiman opened the pull request:
- add footer text
- add footer icon social media

fickryiman committed:
add footer

- add footer text
- add footer icon social media

fickryiman committed:
add hover for get started button and fixes stylelint

MeqdamAlqudah commented on styles.css:
```diff
@@ -405,4 +406,38 @@
+    flex-direction: row;
+    gap: 43px;
+    padding-left: 25px;
+    width: 38px;
```
- [x] Please it would be better to use the grid to design the projects section since the grid is better for two-directional aligning, and containers need that.
![image](https://user-images.githubusercontent.com/74744787/219719952-8c92bf3c-e08c-4b20-9e73-ff8603693c3e.png)

fickryiman replied:
- change projects-section class to use the grid

MeqdamAlqudah commented on index.html:
```diff
@@ -174,6 +174,18 @@ <h2>Skills</h2>
         </div>
```
- [ ] Please the title Projects should use a header tag `hx` since it is semantically better.
![image](https://user-images.githubusercontent.com/74744787/219720894-cda41d2c-6610-4a9f-8686-abc148ad631f.png)
- [ ] Please the project title should use a header tag `hx` since it is semantically better.
![image](https://user-images.githubusercontent.com/74744787/219725652-e7eabd03-f87f-4266-a9f4-ed34b314f037.png)

fickryiman replied:
change all p tag to hx tag

MeqdamAlqudah commented on index.html:
```diff
@@ -174,6 +174,18 @@ <h2>Skills</h2>
         </div>
       </div>
```
- [x] Please it would be better to add hover-on buttons as in the design since it looks better for the users.v
![image](https://user-images.githubusercontent.com/74744787/219726816-dd4d4cf0-01c1-472d-acf8-591859539330.png)

fickryiman replied:
added hover for both skill list and see this project button

MeqdamAlqudah commented on index.html:
```diff
@@ -174,6 +174,18 @@ <h2>Skills</h2>
         </div>
       </div>
       <button type="button" class="button-get-started">Start collaboration</button>
+
```
- [ ] Please this weird space around the project doesn't match the design , suggestions you can consider:-
1 - you can add margin:0 padding: 0 to the body and the html tags .
2- this 0 margin and padding 0 for all elements in the project doesn't help 
![image](https://user-images.githubusercontent.com/74744787/219733436-46e95a0d-4ea8-4272-9717-18e4093dacf0.png)

3- these icons are too large you can modify their size so they don't add much space on the right
![image](https://user-images.githubusercontent.com/74744787/219735113-e4d6a6dd-a786-4316-98f0-318adda920c1.png)
4- please fix the width of these input fields so they don't add space to the right 
![image](https://user-images.githubusercontent.com/74744787/219736426-4dc01dec-5b53-4c0d-840b-1a9bae7d5a93.png)

fickryiman replied:
- change margin 0 and padding 0 from * tags to body, html tags
- resize icon social media
- fixes form width

MeqdamAlqudah commented on index.html:
```diff
@@ -174,6 +174,18 @@ <h2>Skills</h2>
       </div>
       <button type="button" class="button-get-started">Start collaboration</button>
+
+      <footer class="footer">
```
- [ ] Please about myself section title should be wrapped in hx tag since semantically `hx` is used for titles, also note that you should use only the h1 element in the project.

MeqdamAlqudah reviewed (changes requested):
Hi @fickryiman , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
![image](https://user-images.githubusercontent.com/74744787/219749324-bd9cc8b3-c47a-4d1d-8773-bc6c9bd05fc9.png)


## Highlights
:+1: Readme file
:+1: your work section look like the design

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- please test your design more than one time in all sizes
- y
- using responsive design , you can use `vw` in all your design so you can get more responsive sizes

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

fickryiman committed:
Fixes after code review

- change p tags to hx tags
- add hover for skills button and skill list
- change margin and padding is 0 from * to body and html tags
- change social media icon width and height
- fixes form width
- change title section to h1 tags

fickryiman committed:
fixes stylelint

emmiiorji reviewed (approved):
Hi @fickryiman, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

<div align='center'>
<img src='https://user-images.githubusercontent.com/20554583/219607923-71e0205b-a586-4d94-a603-aafadfbe59a4.gif'/>
</div>

## Good Highlights✔️

- Requested changes have been implemented

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
